starter-code/styles/base.css

```diff
@@ -27,7 +27,7 @@ address {
 
 a, a:link, a:active {
   text-decoration: none;
-  background-color: pink;
+  background-color: blue;
```

Comment:
This was a really good find. I'm so glad you found this mistake. You really saved my bacon yet again!